testing/sdk_repository.py

```diff
@@ -96,18 +100,194 @@ def remove_universe_repos(stub_urls):
     log.info('Finished removing universe repos')
 
 
-def universe_session():
+def add_package_registry_stub() -> dict:
+    stub_url = os.environ['PACKAGE_REGISTRY_STUB_URL']
+    with urllib.request.urlopen(stub_url) as url:
+        repo = json.loads(url.read().decode())
+        min_supported = repo["packages"][0]["minDcosReleaseVersion"]
+
+    version = sdk_cmd.dcos_version()
+
+    if not is_repo_supported(version, min_supported):
+        raise Exception('DC/OS {} does not support package registry. '
+                        'Minimum required {}'.format(version, min_supported))
+    return add_stub_universe_urls([stub_url])
+
+
+def install_package_registry(service_secret_path: str) -> dict:
+    # Install Package Registry
+    sdk_install.install(
+        PACKAGE_REGISTRY_NAME,
+        PACKAGE_REGISTRY_SERVICE_NAME,
+        expected_running_tasks=0,
+        additional_options={
+            'registry': {
+                'service-account-secret-path': service_secret_path
+            }
+        },
+        wait_for_deployment=False,
+        insert_strict_options=False
+    )
+
+    pkg_reg_repo = add_stub_universe_urls(
+        ['https://registry.marathon.l4lb.thisdcos.directory/repo']
+    )
+    # If `describe` endpoint is working, registry is writable by AR.
+    package_name = 'hello'
+    package_version = 'world'
+    expected_msg = 'Version [{}] of package [{}] ' \
+                   'not found'.format(package_version, package_name)
+    loop_until_cli_condition(
+        'registry describe --package-name={} --package-version={}'.format(
+            package_name, package_version
+        ),
+        lambda code, out, err: code == 1 and expected_msg in err
+    )
+    return pkg_reg_repo
+
+
+def add_dcos_files_to_registry(
+        temp_dir
```

Comment:
Would it make sense to add type annotations to the parameters as well as the function return types now?